Dockerfile

```diff
@@ -1,22 +1,8 @@
-#ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
-
-# Application image
-
-#FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
-
-#COPY lib/AI-Agent.xml /opt/app/
-#COPY build/libs/spring-boot-template.jar /opt/app/
-
-#EXPOSE 4550
-#CMD [ "spring-boot-template.jar" ]
-
 FROM openjdk:11
 MAINTAINER HMI-QA
 WORKDIR /opt/app/wiremock
 COPY ./wiremock/ /opt/app/wiremock/
-RUN chmod 777 *
+RUN chmod a+x wiremock-jre8-standalone-2.27.1.jar
 RUN ls -al /opt/app/wiremock/
-
 EXPOSE 8080:8080
-#RUN java -jar wiremock-jre8-standalone-2.27.0.jar
-ENTRYPOINT ["java", "-jar", "wiremock-jre8-standalone-2.27.1.jar"]
+ENTRYPOINT ["java", "-jar", "wiremock-jre8-standalone-2.27.1.jar"]
```

Comment:
Missing newline, please be mindful of linting

Reply:
Added a new line at the end of the File.....